### DISCUSSION TITLE
Android Build Failure - PDFbox

### DISCUSSION THREAD

**Discussion by DarkAbhi** [community/community#26937] [Actions]
<p>My project is on a windows machine and it errors out after using the PDFbox library.</p>
<pre><code>&gt; Lint found errors in the project; aborting build.

  Fix the issues identified by lint, or add the following to your build script to proceed with errors:
  ...
  android {
      lintOptions {
          abortOnError false
      }
  }
  ...

  The first 3 errors (out of 10) were:
  D:\a\printola-v3\printola-v3\app\build.gradle: Error: commons-logging defines classes that conflict with classes now provided by Android. Solutions include finding newer versions or alternative libraries that don't have the same problem (for example, for httpclient use HttpUrlConnection or okhttp instead), or repackaging the library using something like jarjar. [DuplicatePlatformClasses]
  C:\Users\runneradmin\.gradle\caches\modules-2\files-2.1\org.apache.pdfbox\fontbox\2.0.0\6f762d4e1c8ea99589d30597ef3731dfdcee43e2\fontbox-2.0.0.jar: Error: Invalid package reference in org.apache.pdfbox:fontbox; not included in Android: java.awt.geom. Referenced from org.apache.fontbox.FontBoxFont. [InvalidPackage]
  C:\Users\runneradmin\.gradle\caches\modules-2\files-2.1\org.apache.pdfbox\pdfbox\2.0.0\c46498f7a734616d0b81e4ea99fd751a17dc34fd\pdfbox-2.0.0.jar: Error: Invalid package reference in org.apache.pdfbox:pdfbox; not included in Android: java.awt.color. Referenced from org.apache.pdfbox.pdmodel.graphics.blend.BlendComposite.BlendCompositeContext. [InvalidPackage]

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.
</code></pre>
<p><strong>Workflow file:</strong></p>
<pre><code>name: Build &amp; Publish Debug APK
on: [push, pull_request]
jobs:
  build:
    runs-on: windows-latest
    steps:
    - uses: actions/checkout@v2
    - name: set up JDK 1.8
      uses: actions/setup-java@v1
      with:
        java-version: 1.8
    - name: Make Gradle executable
      run: chmod +x ./gradlew
    - name: Build with Gradle
      run: ./gradlew build
    - name: Build Debug APK
      run: ./gradlew assembleDebug
    - name: Upload a Build Artifact
      uses: actions/upload-artifact@v2
      with:
        name: Upload app-debug
        path: ./app/build/outputs/apk/debug/app-debug.apk
</code></pre>

**Comment by weide-zhou** [comment, marked as answer]
<p>Hi <a class="mention" href="/darkabhi">@darkabhi</a> ,</p>
<p>The error should be caused by your gradle code but not related to Github Actions. </p>
<p>Please follow below tickets for a try:</p>
<p><a href="https://stackoverflow.com/questions/46989310/commons-logging-defines-classes-that-conflict-with-classes-now-provided-by-andro" rel="nofollow noopener">https://stackoverflow.com/questions/46989310/commons-logging-defines-classes-that-conflict-with-classes-now-provided-by-andro</a></p>
<p><a href="https://github.com/getsentry/sentry-java/issues/571#issuecomment-378295015" rel="nofollow noopener">https://github.com/getsentry/sentry-java/issues/571#issuecomment-378295015</a></p>
<p>It’s recommended to raise the query <a href="https://github.com/gradle/gradle/issues" rel="nofollow noopener">here</a> gradle topic related for more help.</p>
<p>Thanks.</p>
